src/plugins/transform-module/index.ts

```diff
@@ -177,15 +177,14 @@ export default declare((api: any, options: Options) => {
     name: "transform-modules-commonjs",
 
     pre() {
-      // @ts-expect-error
-      this.file.set("@babel/plugin-transform-modules-*", "commonjs");
+      this.set("@babel/plugin-transform-modules-*", "commonjs");
 
-      // @ts-expect-error
-      if (lazy) defineCommonJSHook(this.file, lazyImportsHook(lazy));
+      if (lazy) defineCommonJSHook(this, lazyImportsHook(lazy));
```

Comment:
(TODO: these are runtime changes. Was it essential?)

Reply:
I checked it against the source code from babel, the outcome is the same.

Reply:
I reverted the runtime changes.